lib/Table/TableAutomaticStylesStratum.ts

```diff
@@ -93,8 +92,9 @@ export default class TableAutomaticStylesStratum extends LoadableStratum(
   get defaultChartStyle(): StratumFromTraits<TableStyleTraits> | undefined {
     const scalarColumns = this.catalogItem.tableColumns.filter(
       column =>
-        column.type === TableColumnType.scalar ||
-        column.type === TableColumnType.time
+        !this.catalogItem.excludeStyles?.includes(column.name) &&
```

Comment:
Why is it necessary to add `excludeStyles`? We already have `"type": "hidden"`. Are there situations where you want a column to be a special type (it looks like `excludeStyles` is only relevant if a column is "scalar" or "time") and not have a style for that column?

Reply:
You're right, absolutely not necessary - i have removed it